.changeset/gentle-dancers-greet.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/cli': patch
+---
+
+Added a new ESLint configuration setup for packages, which utilizes package roles to generate the correct configuration. The new configuration is available at `@backstage/cli/config/eslint-factory`.
+
+Introduced a new `backstage-cli migrate package-lint-configs` command, which migrates old lint configurations to use `@backstage/cli/config/eslint-factory`.
```

Comment:
Is this something we should recommend that integrators run for their own plugins?

Reply:
Realise that the PR body mentioned migration as the next step so I believe that answers that :)